test/format/util.c

```diff
@@ -355,6 +356,32 @@ timestamp(void *arg)
     return (WT_THREAD_RET_VALUE);
 }
 
+/*
+ * set_oldest_timestamp --
+ *     Query the oldest timestamp from wiredtiger and set it as our global oldest timestamp. This
+ *     should only be called on runs for pre existing databases.
+ */
+void
+set_oldest_timestamp(void)
+{
+    static const char *oldest_timestamp_str = "oldest_timestamp=";
+
+    WT_CONNECTION *conn;
+    WT_DECL_RET;
+    uint64_t oldest_ts;
+    char buf[WT_TS_HEX_STRING_SIZE * 2 + 64], tsbuf[WT_TS_HEX_STRING_SIZE];
+
+    conn = g.wts_conn;
+
+    if ((ret = conn->query_timestamp(conn, tsbuf, "get=oldest")) == 0) {
+        timestamp_parse(NULL, tsbuf, &oldest_ts);
+        g.timestamp = oldest_ts;
+        testutil_check(
+          __wt_snprintf(buf, sizeof(buf), "%s%" PRIx64, oldest_timestamp_str, g.oldest_timestamp));
+    } else
+        testutil_die(ret, "unable to query oldest timestamp");
```

Comment:
Does this call from format returns error or none when the timestamps are disabled in the config?

Reply:
See latest commit.

Reply:
Thanks for explaining and fix.